### PR TITLE
Aug 595/bullet chart

### PR DESCRIPTION
**Pull request by bakesaled**
Story: https://jira.inbcu.com/browse/AUG-595

### CODE REVIEW

File: src/bullet-chart.component.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'supre-bullet-chart',
+  templateUrl: './bullet-chart.component.html',
+  styleUrls: ['./bullet-chart.component.scss']
```
Comment (bcandullo) [inline src/bullet-chart.component.ts]: is there a reason change detection strategy is default? in my experience getters + default change detection is very expensive. if you log a getter you can see it in action. nice job.

Comment (bakesaled) [inline src/bullet-chart.component.ts]: Just an oversight on my part.  Good catch!

**Comment by paulkahle** [comment]
@bakesaled 
In general this looks good.  There is one issue with the barWith calculation when minWith > 0. The formula should be Math.max(value - minValue, 0)/(maxValue - minValue).  
1. You have to subtract minValue from value in the calculation.  Eg. When minValue = 100 and value = 100, the bar width should be 0.
2. Math.max(value - minValue, 0) will prevent a negative width if value < minValue.

File: src/bullet-chart.component.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+   */
+  @Input()
+  public comparisonValue: number;
+
+  /**
+   * Width of the bar as a percentage based on the value.
+   * The width should not be greater than the maxValue.
+   * @returns {number}
+   */
+  public get barWidth() {
```
Comment (paulkahle) [inline src/bullet-chart.component.ts]: (Math.max(this.value - this.minValue, 0)/(this.maxValue - this.minValue)) * 100

**Comment by bakesaled** [comment]
Thanks @paulkahle   It looked like the same formula needed to be applied to the comparisonValue as well.

**Pull request merged**